les/tokensale.go

```diff
@@ -593,12 +581,8 @@ func (t *tokenSale) getBalance(id enode.ID) (pcBalance, tokenBalance uint64) {
 	defer t.lock.Unlock()
 
 	pb := t.clientPool.getPosBalance(id)
-	tokenBalance = pb.value.value(t.clientPool.posExpiration(mclock.Now()))
-	var meta tokenSaleMeta
-	if err := rlp.DecodeBytes([]byte(pb.meta), &meta); err == nil {
-		pcBalance = meta.pcBalance
-	}
-	return
+	cb := t.clientPool.ndb.getCurrencyBalance(id)
+	return pb.value.value(t.clientPool.posExpiration(mclock.Now())), cb.amount
```

Comment:
It should be `cb.amount, pb.value.value(t.clientPool.posExpiration(mclock.Now()))`. Will be fixed by latter commits.